client/src/Views/Reports/MonthlyReports.vue

```diff
@@ -1,15 +1,17 @@
 <template>
     <v-tab-item>
-          <v-date-picker v-model="picker" type="month" color="green darken-1">
-          </v-date-picker>
-          <Table
-            class="reports-container__table"
-            :data="shoutsToDisplay"
-            @toggleView="toggleView"
-            :view="tableView"
-            :dateObj="this.createDateObj()"
-    />
-    <RadarChart :data="shoutsToDisplay.slice(0, 7)" />
+      <div class="item-container">
+        <v-date-picker v-model="picker" type="month" color="#1aa5ca" class="item-container__picker">
```

Comment:
Should probably use the color as a variable...

Reply:
Good call. I'm also going to merge your PR and incorporate them into my changes to avoid merge conflicts.